main.py

```diff
@@ -2666,59 +2666,23 @@ async def callback_hell(interaction, thing):
-        # for i in ["Cat Hunt", "Random", "Unfair"]:
-        #   if category == i:
-        #        buttons_list.append(Button(label=i, style=ButtonStyle.green))
-        #   else:
-        #        buttons_list.append(Button(label=i, style=ButtonStyle.blurple))
-        #   lambdas_list.append(lambda interaction : (await interaction.edit_original_response(embed=gen_new(i), view=insane_view_generator(i)) for _ in '_').__anext__())
-        #   buttons_list[-1].callback = lambdas_list[-1]
-
-        if category == "Cat Hunt":
-            buttons_list.append(Button(label="Cat Hunt", style=ButtonStyle.green))
-        else:
-            buttons_list.append(Button(label="Cat Hunt", style=ButtonStyle.blurple))
-        lambdas_list.append(lambda interaction : (await callback_hell(interaction, "Cat Hunt") for _ in '_').__anext__())
-        buttons_list[-1].callback = lambdas_list[-1]
-
-        if category == "Random":
-            buttons_list.append(Button(label="Random", style=ButtonStyle.green))
-        else:
-            buttons_list.append(Button(label="Random", style=ButtonStyle.blurple))
-        lambdas_list.append(lambda interaction : (await callback_hell(interaction, "Random") for _ in '_').__anext__())
-        buttons_list[-1].callback = lambdas_list[-1]
-
-        if category == "Silly":
-            buttons_list.append(Button(label="Silly", style=ButtonStyle.green))
-        else:
-            buttons_list.append(Button(label="Silly", style=ButtonStyle.blurple))
-        lambdas_list.append(lambda interaction : (await callback_hell(interaction, "Silly") for _ in '_').__anext__())
-        buttons_list[-1].callback = lambdas_list[-1]
-
-        if category == "Hard":
-            buttons_list.append(Button(label="Hard", style=ButtonStyle.green))
-        else:
-            buttons_list.append(Button(label="Hard", style=ButtonStyle.blurple))
-        lambdas_list.append(lambda interaction : (await callback_hell(interaction, "Hard") for _ in '_').__anext__())
-        buttons_list[-1].callback = lambdas_list[-1]
-
-        if category == "Hidden":
-            buttons_list.append(Button(label="Hidden", style=ButtonStyle.green))
-        else:
-            buttons_list.append(Button(label="Hidden", style=ButtonStyle.blurple))
-        lambdas_list.append(lambda interaction : (await callback_hell(interaction, "Hidden") for _ in '_').__anext__())
-        buttons_list[-1].callback = lambdas_list[-1]
+        for i in ["Cat Hunt", "Random", "Silly", "Hard", "Hidden"]: # hear me out what if it works :picardia_shrug:
+            if category == i:
+                buttons_list.append(Button(label=i, style=ButtonStyle.green))
+            else:
+                buttons_list.append(Button(label=i, style=ButtonStyle.blurple))
+            lambdas_list.append(lambda interaction : (await callback_hell(interaction, i) for _ in '_').__anext__())
+            buttons_list[-1].callback = lambdas_list[-1]
```

Comment:
how about no???

Reply:
awh its been sitting like that in the code for- hold on lemme check

Reply:
github says yesterday but thats just because github thought you changed it yesterday because it loves being broken

Reply:
that comment sat there since about the time /achievements was added

Reply:
and i currently have no reason to believe it will magically work after years with no changes to it

Reply:
yk usually old stuff breaks over time not in reverse

Reply:
no i did change it lol

Reply:
before it didn't have a defer in it so it wasn't going to work